app-header/app-header.html

```diff
@@ -116,9 +111,7 @@
       opacity: 0;
       box-shadow: inset 0px 5px 6px -3px rgba(0, 0, 0, 0.4);
       will-change: opacity;
-      -webkit-transition: opacity 0.4s;
       transition: opacity 0.4s;
-      -webkit-transform: translateZ(0);
```

Comment:
Did you find any perf/rendering implication in Safari 8, 9 and iOS?  I remember seeing a few issues that went away after adding this rule `-webkit-transform: translateZ(0);`. In one of them, the shadow didn't render properly and it looked like a solid black box (clearly a bug in Safari). In the second issue, I noticed that sometimes the shadow rendered underneath the content because we don't use z-index, so adding `translateZ(0)` I believe fixed the issue. It would be good to try with a fixed header that doesn't condense, but that has the waterfall effect to see if we can still repro the issue.